saleor/graphql/payment/resolvers.py

```diff
@@ -16,6 +24,26 @@ def resolve_payments(info, query):
 
 def resolve_payment_client_token(gateway=None, user=None):
     token_config = TokenConfig()
-    if user:
+    if user is not None:
         token_config.customer_id = extract_id_for_payment_gateway(user, gateway)
     return gateway_get_client_token(gateway, token_config)
+
+
+def resolve_payment_sources(user):
+    def retrieve_sources(gateway_name, customer_id):
```

Comment:
I would think it would be better to put it outside of the function, with a more meaningful name. It seems like the logic could be used by other functions. Maybe but it private?

Reply:
Moved to utils